Orange/util.py

```diff
@@ -3,20 +3,25 @@
 from functools import wraps
 from itertools import chain, count
 from collections import OrderedDict
-import logging
+import warnings
 
 import numpy as np
 
 
-log = logging.getLogger()
+warnings.simplefilter('default', DeprecationWarning)
```

Comment:
Yes. Someone might setup an elaborate warning filter rules (e.g. on subclasses of DeprecationWarning, only raised in some specific module, ...), which would be overridden simply by importing Orange.